src/Service/Notification.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\MFA\Service;
+
+use Exception;
+use Psr\Log\LoggerInterface;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Core\Extensible;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Security\Member;
+
+/**
+ * Ecapsulates setting up an Email in order to allow for Dependency Injection and to avoid introducing a hard
+ * coupling to the SilverStripe core Email class in code that consumes this class.
+ */
+class Notification
```

Comment:
Did you want to rename the class? The reason I suggested it because `$dependencies = ['NotificationService' => Notification::class]` is not as verbose as `NotificationService::class` despite it being namespaced. If you think it's fine then I'm happy too. You could also use `Notification\Service::class`, or leave it as is. The property name helps 👍